### PR TITLE
update Traditional Chinese translation

### PR DESCRIPTION
**Pull request by 5idereal**

### CODE REVIEW

File: packages/core/content-type-builder/admin/src/translations/zh.json [hunk 1]
```diff
@@ -36,19 +39,31 @@
   "button.model.create": "建立新的集合類型",
   "button.single-types.create": "建立新的單一類型",
   "component.repeatable": "(可重複的)",
+  "components.SelectComponents.displayed-value": "{number, plural, =0 {# components} one {# component} other {# components}} selected",
```
Comment (ronronscelestes) [inline packages/core/content-type-builder/admin/src/translations/zh.json:42]: I think this one is missing its translation? 👀

File: packages/core/content-type-builder/admin/src/translations/zh.json [hunk 2]
```diff
@@ -129,24 +159,39 @@
...
   "modalForm.sub-header.chooseAttribute.component": "選擇元件的一個欄位",
   "modalForm.sub-header.chooseAttribute.singleType": "選擇單一型別的一個欄位",
+  "modalForm.custom-fields.advanced.settings.extended": "擴充設定",
+  "modalForm.tabs.custom": "自訂",
+  "modalForm.tabs.custom.howToLink": "如何新增自訂欄位",
+  "modalForm.tabs.default": "預設",
+  "modalForm.tabs.label": "預設和自訂類型分頁",
   "modelPage.attribute.relation-polymorphic": "關聯 (多對多多型)",
   "modelPage.attribute.relationWith": "關聯到",
+  "notification.error.dynamiczone-min.validation": "At least one component is required in a dynamic zone to be able to save a content type",
```
Comment (ronronscelestes) [inline packages/core/content-type-builder/admin/src/translations/zh.json:169]: this one too :)

File: packages/plugins/i18n/admin/src/translations/zh.json [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+  "CheckboxConfirmation.Modal.button-confirm": "是的，請停用",
+  "CheckboxConfirmation.Modal.content": "Disabling localization will engender the deletion of all your content but the one associated to your default locale (if existing).",
+  "Field.localized": "This value is unique for the selected locale",
+  "Field.not-localized": "This value is common to all locales",
+  "Settings.list.actions.add": "新增地區設定",
+  "Settings.list.actions.delete": "刪除地區設定",
+  "Settings.list.actions.deleteAdditionalInfos": "This will delete the active locale versions <em>(from Internationalization)</em>",
+  "Settings.list.actions.edit": "編輯地區設定",
+  "Settings.list.description": "設定國際化外掛程式",
+  "Settings.list.empty.description": "This is not a usual behavior, meaning that you have eventually modified the database manually. Make sure to have at least one locale saved in your database in order to be able to use Strapi correctly.",
```
Comment (ronronscelestes) [inline packages/plugins/i18n/admin/src/translations/zh.json:17]: this one too

**Review by ronronscelestes — CHANGES_REQUESTED** [review]
hi @5idereal, thank you for those great translation improvements!
I highlighted 3 translations key are missing their Chinese translations (they are still in English) 🙂

**Comment by 5idereal** [comment]
Thanks for the review! I've updated the translation files.

**Review by ronronscelestes — APPROVED** [review]
thank you for the fixes @5idereal :)
I'll merge your PR once the tests are ✅

**Pull request merged**